tests/src/org.hibernate.orm/hibernate-core/6.5.0.Final/src/test/resources/hibernate.properties

```diff
@@ -0,0 +1,2 @@
+hibernate.bytecode.provider=none
```

Comment:
Can this file be deleted since these properties are not read in this version of hibernate?

Reply:
yeah, sorry - overlooked that one